Sources/Sampler/custom_sampler.hpp

```diff
@@ -101,6 +103,13 @@ class CustomSampler : public AbstractSampler {
     nstates_ = GetHilbert().LocalSize();
     localstates_ = GetHilbert().LocalStates();
 
+    // Always use odd sweep size to avoid possible ergodicity problems
+    if (nv_ % 2 == 0) {
```

Comment:
This can be written somewhat shorter as `SetSweepSize(nv_ + (nv_ % 2 == 0))`, but if you find the current version easier to read, let's stick to it.

Reply:
Yes, let's stick to it